plugins/Dbx_mdbx/src/version.h

```diff
@@ -5,9 +5,9 @@
 
 #include <stdver.h>
 
-#define __PLUGIN_NAME            "Miranda NG LMDB database driver"
-#define __FILENAME               "Dbx_mdb.dll"
+#define __PLUGIN_NAME            "Miranda NG MDBX database driver (https://github.com/leo-yuriev/libmdbx)"

```

Comment:
I think we should mention this link into wiki, not in plugin description

Reply:
I agree.